internal/util/funcutil/func.go

```diff
@@ -360,6 +360,14 @@ func IsGrpcErr(err error) bool {
 	if err == nil {
 		return false
 	}
-	_, ok := grpcStatus.FromError(err)
-	return ok
+	for {
+		_, ok := grpcStatus.FromError(err)
```

Comment:
It may be better to move the nil check at 369 line here, and remove the nil check at 360 line.

Reply:
Fixed